core/sdk-core/src/main/java/software/amazon/awssdk/core/sync/RequestBody.java

```diff
@@ -49,11 +50,10 @@
     private final long contentLength;
```

Comment:
Can we change it to `Long contentLength`? That way, we can just check null instead in the `contentLength` method